packages/near-fast-auth-signer/webpack.config.js

```diff
@@ -39,15 +42,25 @@ module.exports = {
       FIREBASE_APP_ID_TESTNET:              '1:116526963563:web:053cb0c425bf514007ca2e',
       FIREBASE_MEASUREMENT_ID_TESTNET:      'G-HF2NBGE60S',
       SENTRY_DSN:                           'https://1049553ebca8337848160ca53a49ff2a@o398573.ingest.sentry.io/4506148066164736',
-    })
+      SENTRY_DSN_TESTNET:                   'https://ce94b1ec626e971719c20fa7979158f3@o398573.ingest.sentry.io/4506702275411968',
+    }),
+    ...(process.env.SENTRY_AUTH_TOKEN
+      ? [sentryWebpackPlugin({
+        org:       'near-protocol',
+        project:   process.env.NETWORK_ID === 'mainnet' ? 'fast-auth-signer' : 'fast-auth-signer-testnet',
+        authToken: process.env.SENTRY_AUTH_TOKEN,
+        include:   './fastauth',
+      })]
+      : []
+    ),
   ],
   devServer: {
     static: {
       directory: path.join(__dirname, 'dist'),
     },
     port: 3000,
   },
-  devtool: 'eval-source-map',
+  devtool: 'source-map',
```

Comment:
Isn't better to run the `eval-source-map` for dev as it's faster and `source-map` for prod? 

The `eval-source-map` it's faster and recommended for dev: https://webpack.js.org/configuration/devtool/

<img width="739" alt="Screenshot 2024-02-07 at 5 18 53 PM" src="https://github.com/near/fast-auth-signer/assets/22358004/ed9d5c65-3434-490a-a557-6938f9a7daa5">